zkmemory/src/commitment/kzg.rs

```diff
@@ -136,17 +140,16 @@ where
     // This is made compatible with the Fr endianess
     fn be_bytes_to_field(&self, bytes: &mut [u8]) -> Fr {
         bytes.reverse();
-        //let b = bytes.as_ref();
         let inner = [0, 8, 16, 24].map(|i| u64::from_le_bytes(bytes[i..i + 8].try_into().unwrap()));
```

Comment:
Should not `unwrap()` without reason, it's terrible.
```suggestion
        let inner = [0, 8, 16, 24].map(|i| u64::from_le_bytes(bytes[i..i + 8].try_into().expect("Unable to convert data to u64")));
```